modules/proto/src/smithyproto/proto3/ModelPreProcessor.scala

```diff
@@ -1,308 +0,0 @@
-/* Copyright 2022 Disney Streaming
- *
- * Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    https://disneystreaming.github.io/TOST-1.0.txt
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package smithyproto.proto3
-
-import java.util.stream.Collectors
-import smithytranslate.closure.TransitiveModel
-import smithytranslate.UUID
-import software.amazon.smithy.build.{ProjectionTransformer, TransformContext}
-import software.amazon.smithy.model.Model
-import software.amazon.smithy.model.loader.Prelude
-import software.amazon.smithy.model.shapes._
-
-import java.util
-import scala.jdk.CollectionConverters._
-import scala.collection.compat._
-
-object ModelPreProcessor {
```

Comment:
removed this as a lot of what it was doing is now un-necessary. The logic for guarding against conflicts in enum values moved to the Compiler